api/webhooks/sample_webhook_data.py

```diff
@@ -0,0 +1,61 @@
+environment_webhook_data = {
```

Comment:
We should obfuscate / anonymise this data, e.g. don't use Ben / Kyle's names, don't use our actual butter bar feature flag, maybe update some of the ids.

Reply:
I made a couple of extra minor tweaks to be particularly picky but this is good now.